src/App.css

```diff
@@ -6,6 +6,56 @@
   padding: 0;
 }
 
+body {
+  background-color: #ecf0f3;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
 .App {
+  font-family: sans-serif;
   text-align: center;
+  border: none;
+  height: 640px;
+  width: 360px;
+  max-width: 700px;
+  border-radius: 40px;
+  box-shadow: 13px 13px 20px #cbced1, -13px -13px 20px #ffffff;
+  background-color: #ecf0f3;
+  display: grid;
+  grid-template-rows: [header] 100px [content] 480px [footer] 60px;
+  margin: 20px 0;
+}
+
+header {
```

Comment:
Maybe I'd add a class here instead of styling the element directly. Same for the others.